manifests/crd.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
I suggest to move these manifest to a separate PR and then let's concentrate on the test case

Reply:
These manifest actually helps us easily deploy tf-operator but it's unrelated to test itself.

Reply:
I have removed the manifests from this PR
@Jeffwan Does it mean that the scope of this PR is just to make sure it can trigger the testing CI 
and then we will have another PR for the testing content?

Reply:
@ChanYiLin  can we reuse existing test cases? I know the way to deploy tf-operator is different now.  Besides that, all changes in this PR should be testing related.

Reply:
@Jeffwan I believe the e2e test cases has already been there and after the deployment it will then run the test case
for example [link](https://github.com/kubeflow/tf-operator/pull/1204/files#diff-d99e924ca9f84f28b6c4decf07cee8a86513c5852272b05512dca7081d3e9e75R392)
The reason I added the manifest is because it is more easily to deploy tf-operator since the originally way is to use ksonnet and python script for example [link](https://github.com/kubeflow/tf-operator/pull/1204/files#diff-d99e924ca9f84f28b6c4decf07cee8a86513c5852272b05512dca7081d3e9e75L364)
I believe this is also the way how pytorch setup the cluster for testing?
Is there anything I misunderstood?

Reply:
Got you. Make sense. Let's do this way. My original thought is to split manifest change and test infra change. This is not a big deal.